Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/UnifiedPDFPlugin.mm

```diff
@@ -1065,10 +1074,15 @@ static AffineTransform documentSpaceToPageSpaceTransform(const IntDegrees& pageR
 
         switch (auto mouseEventButton = event.button()) {
         case WebMouseEventButton::Left:
-            if (m_annotationTrackingState.trackedAnnotation()) {
+            if (auto trackedAnnotation = m_annotationTrackingState.trackedAnnotation()) {
```

Comment:
We might want to use `RetainPtr` here or swap the order of operations below since `finishAnnotationTracking` clears the annotation it is holding onto

Reply:
or maybe change the API of `trackedAnnotation()`